cmd/acra-keys/keys/migrate-keys.go

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright 2020, Cossack Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package keys
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/cossacklabs/acra/cmd"
+	keystoreV1 "github.com/cossacklabs/acra/keystore"
+	filesystemV1 "github.com/cossacklabs/acra/keystore/filesystem"
+	keystoreV2 "github.com/cossacklabs/acra/keystore/v2/keystore"
+	keystoreAPIV2 "github.com/cossacklabs/acra/keystore/v2/keystore/api"
+	filesystemV2 "github.com/cossacklabs/acra/keystore/v2/keystore/filesystem"
+	log "github.com/sirupsen/logrus"
+)
+
+// MigrateKeysParams ara parameters of "acra-keys migrate" subcommand.
+type MigrateKeysParams interface {
+	SrcKeyStoreVersion() string
+	SrcKeyStoreParams() KeyStoreParameters
+	DstKeyStoreVersion() string
+	DstKeyStoreParams() KeyStoreParameters
+	DryRun() bool
+	ForceWrite() bool
+}
+
+// MigrateKeysSubcommand is the "acra-keys migrate" subcommand.
+type MigrateKeysSubcommand struct {
+	flagSet    *flag.FlagSet
+	src, dst   CommonKeyStoreParameters
+	srcVersion string
+	dstVersion string
+	dryRun     bool
+	force      bool
+}
+
+// Environment variables from which master keys are read.
+const (
+	SrcMasterKeyVarName = "SRC_" + keystoreV1.AcraMasterKeyVarName
+	DstMasterKeyVarName = "DST_" + keystoreV1.AcraMasterKeyVarName
+)
+
+// Command-line errors for "acra-keys migrate":
+var (
+	ErrMissingFormat = errors.New("key store format not specified")
```

Comment:
```suggestion
	ErrMissingFormat = errors.New("keystore format not specified")
```
not sure which wording we use